README.md

```diff
@@ -1,5 +1,4 @@
-NDLA H5P Interactive Board
-==========
+# NDLA H5P Interactive Board
```

Comment:
Oh, does `prettier` also work on markdown? Nice. Didn't know that. (just a note :-))